checkov/cloudformation/graph_builder/variable_rendering/renderer.py

```diff
@@ -26,13 +37,13 @@ class CloudformationVariableRenderer(VariableRenderer):
 
     def __init__(self, local_graph: "CloudformationLocalGraph") -> None:
         super().__init__(local_graph)
-        self.edge_evaluation_methods = {
+        self.edge_evaluation_methods: "dict[str, Callable[[Any, dict[str, Any]], tuple[str | None, str | None]]]" = {
```

Comment:
perhaps create a `TypeVar` here and move to the top of the module?

Reply:
I actually used a `TypeAlias`, that's what you mean, right?, initially and changed that line so often 😄 sure , can bring it back